TicketingSystem.Common/Context/TicketingDbContext.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using System.Reflection.Metadata;
+using TicketingSystem.Common.Model.Database;
+using TicketingSystem.Common.Model.Database.Enums;
+
+namespace TicketingSystem.Common.Context
+{
+    public class TicketingDbContext(DbContextOptions<TicketingDbContext> options) : DbContext(options), ITicketingDbContext
+    {
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            base.OnModelCreating(modelBuilder);
+
+            modelBuilder.Entity<Ticket>()
```

Comment:
Added configurations for all entities
I only let efcore configure the many-to-many relationship between Carts and Tickets by convention because when I tried to do it in config the table CartsTickets didn't create

Verified that the migration is the same with or without configs